google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineCloudConfigurable.java

```diff
@@ -35,17 +39,40 @@
   private static final String MORE_INFO_URI_OPEN_TAG =
       "<a href='https://cloud.google.com/appengine/'>";
   private static final String MORE_INFO_URI_CLOSE_TAG = "</a>";
+  private static final String PSEUDO_GOOGLE_SDK_LINK = "http://google-sdk/";
 
   private String displayName = GctBundle.message("appengine.name");
   private JPanel mainPanel;
   private JTextPane appEngineMoreInfoLabel;
-  private CloudSdkPanel cloudSdkPanel;
 
   /** Initialize the UI. */
-  public AppEngineCloudConfigurable() {
-    appEngineMoreInfoLabel.setText(
+  AppEngineCloudConfigurable() {
+    // in case managed SDK is not available yet, provide direct link to SDK settings to allow
```

Comment:
Ah. I guess this almost answers my other question: can you maybe show screenshots of examples of when this link is shown (and any warning messages in case the custom sdk is misconfigured)?

Reply:
Sure, attaching screenshots.